unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/IFriendsController.cs

```diff
@@ -54,5 +54,6 @@ public interface IFriendsController
         void GetFriendsWithDirectMessages(string userNameOrId, int limit);
         SocialFriendRequest GetAllocatedFriendRequest(string friendRequestId);
         SocialFriendRequest GetAllocatedFriendRequestByUser(string userId);
+        UniTask<FriendshipStatus> GetFriendshipStatus(string userId);
```

Comment:
I propose `GetUpdatedFriendshipStatus `, since the logic inside class not only getting the status from API, but also Updates it before passing the result

Reply:
Internally updating the cached status is an implementation detail, not an interface design, nor a standard behaviour

Reply:
Good point. Closing